src/components/ui/EditorPreview.tsx

```diff
@@ -23,7 +23,7 @@ export interface EditorPreviewRef {
 export const EditorPreview = forwardRef<EditorPreviewRef, EditorPreviewProps>(
   ({ className, content }, ref) => {
     const [html, setHtml] = useState("")
-    const [visible, setVisible] = useState(false)
+    const [visible, setVisible] = useState(true)
```

Comment:
what about getting `visible` and `setVisible` from props? since you already delcared `previewVisible` and `setPreviewVisble` in `editor.tsx` 🤔